### PR TITLE
#166239312 Set up Heroku deployment

### PR DESCRIPTION
**Pull request by Emmanuel-Dominic**
#### What does this PR do?
 - setup and deploy Heroku plus enabling review apps

#### Description of Task to be completed?
 - Currently, we do not have any deployment of the application. There is a need to deploy the current state of the application on Heroku so as to monitor progress and have visibility of the working system.

#### How should this be manually tested?
 - Simply click on this link to access the Heroku application URL [Heroku Link](https://wger-kronos.herokuapp.com)

#### What are the relevant pivotal tracker stories?
- [#166239312](https://www.pivotaltracker.com/n/projects/2350327/stories/166239312)

### CODE REVIEW

**Comment by Rhytah** [comment]
@Emmanuel-Dominic  could you consider editing the PR description, especially `What does this PR do?` to be more specific to the story scope of the task at hand.

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks for the work done @Emmanuel-Dominic, kindly include the links in the PR description for us to test the deployments you have set up. In addition, make your story id linkable to pivotal tracker.

**Review by Manorlds-Eaglespark — COMMENTED** [review]
Thanks for this great work so far, however, to approve your changes, I will wait till all the tests are passing on Travis.

**Comment by AGMETEOR** [comment]
@Emmanuel-Dominic change the description to try and match the story description on the PT Board. Also, link the story id to the PT story.

File: settings.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.postgresql',
+#         'NAME': os.environ.get('NAME'),
+#         'USER': os.environ.get('USER'),
+#         'PASSWORD': os.environ.get('PASSWORD'),
+#         'HOST': os.environ.get('HOST'),
+#         'PORT': os.environ.get('PORT'),
+#     }
+# }
```
Comment (AGMETEOR) [inline settings.py]: Is this commented out code necessary?

**Pull request closed without merging**